LoopKit/Notification/LoopNotificationUserInfoKey.swift

```diff
@@ -9,5 +9,6 @@
 public enum LoopNotificationUserInfoKey: String {
     case bolusAmount
     case bolusStartDate
-    case cgmAlertID
+    case alertID
```

Comment:
Note: I've started wiring up the ability to acknowledge from UserNotifications, but Design has not finalized for this yet.  So for now it is disabled, and all you can do is open the app.